lib/bookclub.rb

```diff
@@ -0,0 +1,40 @@
+class BookClub
+  attr_reader :books
+
+  def initialize
+    @books = []
+  end
+
+  def add_book(book)
+    @books.push(book)
+  end
+
+  def unread_books
+    books.reject do |book|
+      book.read?
+    end
+  end
+
+  def read_books
+    books.select do |book|
+      book.read?
+    end
+  end
+
+  def read_book(book)
+    book.read = true
+  end
+
+  def total_reviews
+    @total_reviews = 0
```

Comment:
We don't need to use this as an instance variable, we can just use `sum` and just return the value